src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -30,6 +30,8 @@ public class ModelManager implements Model {
     private final UserPrefs userPrefs;
     private final FilteredList<Person> filteredPersons;
     private final FilteredList<Person> loggedFilteredPersons;
+    private String lastExecutedCommandType = "unknown";
```

Comment:
What would this `String` do? It doesn't seem to be referenced anywhere else, and this doesn't seem relevant to the `log` command.

Reply:
Good catch, I had added it when pursuing an alternative implementation, and it slipped my eye. It will be removed in the next commit.